bot/exts/info/resources.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional
+from urllib.parse import quote
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+
+class Resources(commands.Cog):
+    """Display information about the Python Discord website Resource page."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(name="resources", aliases=("res",))
+    async def resources_command(self, ctx: commands.Context, *, resource_topic: Optional[str]) -> None:
+        """Display information and a link to the Python Discord website Resources page."""
+        # If resource_topic isn't provided, provide link to the resources' page without any filtering of topics
+        if not resource_topic:
+            embed = Embed(
+                title="Resources",
+                description="The [Resources page](https://www.pythondiscord.com/resources) on our "
+                            "website contains a list of hand-selected learning resources that we "
+                            "regularly recommend to both beginners and experts."
+            )
+
+            await ctx.send(embed=embed)
+            return
+
+        # Capture everything prior to the new line allowing users to add messages below the command then prep for url
+        resource_topic = resource_topic.splitlines()[0].lower().replace(" ", "-")
+        resource_url = f"https://www.pythondiscord.com/resources/?topics={quote(resource_topic)}"
+
+        embed = Embed(
+            title="Resources",
+            description=f"The [Resources page]({resource_url}) on our website contains a list "
+                        f"of hand-selected learning resources that we "
+                        f"regularly recommend to both beginners and experts."
+        )
+        await ctx.send(embed=embed)
```

Comment:
We can actually simplify this and repeat less code:
```python
        resource_url = "https://www.pythondiscord.com/resources/"

        if resource_topic:
            # Capture everything prior to the new line allowing users to add messages below the command then prep for url
            resource_topic = resource_topic.splitlines()[0].lower().replace(" ", "-")
            resource_url = f"{resource_url}?topics={quote(resource_topic)}"

        embed = Embed(
            title="Resources",
            description=f"The [Resources page]({resource_url}) on our website contains a list "
                        f"of hand-selected learning resources that we "
                        f"regularly recommend to both beginners and experts."
        )
        await ctx.send(embed=embed)
```

Reply:
Note that I think that `resource_url` could be changed to `RESOURCE_URL` and be a constant at the top, then we define the same variable in the function (`url = RESOURCE_URL` and then `url = f"{url}?topics={...}"`)